charts/ai-software-templates/chatbot/templates/route.yaml

```diff
@@ -15,5 +14,5 @@ spec:
   to:
     kind: Service
     name: {{ .Release.Name }} 
-    weight: 100
```

Comment:
nit: no need to add the space at the end of the line, right?

Reply:
I think is related to https://github.com/redhat-ai-dev/ai-lab-helm-charts/pull/31#discussion_r1923635863

Reply:
gotcha - ok